modelzoo/TwitterRecSys2021/README.md

```diff
@@ -1,24 +1,46 @@
-# Intel Democratized Solutions for RecSys Challenge 2021 
+# Intel Optimized Solutions for RecSys Challenge 2021 
 
-# How to Use
+# RecSys Challenge 2021 
+The challenge focuses on a real-world task of tweet engagement prediction in a dynamic environment. Please check the [website](https://recsys.acm.org/recsys21/challenge/) on the details to get the dataset. 
+
+# Quick Start
 
 ## Prepare
-1. Environment
-    * Spark 3.0
-    * Hadoop 3.2
-    * lightgbm 3.2.1
-    * XGBoost 1.3.3
-    * transformers 4.13.0
-2. Prepare 
-    * install pyrecdp
-    ```bash
-    pip install pyrecdp
-    ```
-    * Put original dataset(train, valid, test, valid_split_index) at HDFS: /recsys2021/oridata
-    * Create two folds in HDFS to save data for stage1 and stage2: /recsys2021/datapre_stage1/, /recsys2021/datapre_stage2/
+```
+export path_to_e2eaiok_dataset=`pwd`/e2eaiok_dataset # put the downloaded dataset here
+export path_to_e2eaiok=`pwd`/e2eAIOK
+git clone https://github.com/intel/e2eAIOK.git
+git submodule update --init --recursive
+```
+
+## Environment setup
+You can choose to use AIOK docker or prepare the environment by yourself.
+
+### User AIOK Docker
+```
+cd ${path_to_e2eaiok}/Dockerfile-ubuntu18.04/
+docker build -t e2eaiok-pytorch-spark . -f DockerfilePytorch-spark
+cd ${path_to_e2eaiok}
+docker run --shm-size=10g -it --privileged --network host -v ${path_to_e2eaiok_dataset}:/home/vmagent/app/dataset -v `pwd`/:/home/vmagent/app/e2eaiok -w /home/vmagent/app/ e2eaiok-pytorch-spark /bin/bash
+source /etc/profile.d/spark-env.sh
+pip install pyrecdp
+Install hadoop and update the setting
+```
+
+### Prepare environment by yourself
+Install the following libs:
+ * Spark 3.0
+ * Hadoop 3.2
+ * lightgbm 3.2.1
+ * XGBoost 1.3.3
+ * transformers 4.13.0
+ * pyrecdp
+   
```

Comment:
with DockerfilePytorch, you should only need to install pyrecdp

Reply:
have removed this~